guidelines.md

```diff
@@ -1,32 +1,52 @@
 # Translation guidelines
 
-There are some points we would like you to keep in mind when writing translation.
+There are some points we would like you to keep in mind while working on your translations.
 
 ### Friendly, cheerful and positive language
 
+<<<<<<< HEAD
```

Comment:
Graaah, I hate merges! That probably happened because I had the files open in both my default merge tool and Sublime, gnah. Will go and fix it, thanks for making me aware. (: